### PR TITLE
Update to home page to contribute towards issue #87

### PR DESCRIPTION
**Pull request by jorgebustamante**
This updates text styling and mobile responsiveness to more accurately represent the live org website.
This should meet 3/4 of the criteria for completing the issue, just need to hook up email now.  

## Before
Screen Shot 2020-04-23 at 6 32 29 PM

## After
Screen Shot 2020-04-23 at 6 45 01 PM

I additionally brought in react-icons to solve the outgoing links in regards to the connect with us section

Screen Shot 2020-04-23 at 7 01 19 PM
To get started:

- Install dependencies

```
yarn install
```

## Test

Make sure these commands work and are successful:

- Development 

```
yarn start
```

- Build

```
yarn build
```

- Export

```
yarn export
```

- Storybook

```
yarn storybook
```

### CODE REVIEW

**Comment by arecvlohe** [comment]
Looking good! I will look at it this weekend!

File: pages/index.tsx [hunk 1]
```diff
@@ -71,9 +73,63 @@ export default function Index() {
...
+            {
+              icon: AiFillTwitterCircle,
+              url: 'https://twitter.com/nativesintech'
+            },
+            {
+              icon: FaLinkedin,
+              url: 'https://www.linkedin.com/company/natives-in-tech'
+            }
+          ].map(iconLink => (
+            <a className='' href={iconLink.url} key={iconLink.icon.toString()}>
```
Comment (arecvlohe) [inline pages/index.tsx:94]: Use `url` for `key`, that's unique enough

Comment (arecvlohe) [inline pages/index.tsx:94]: Remove className since it's empty

**Comment by jorgebustamante** [comment]
Thanks for the feedback! I’ll hop on fixing these tonight

**Pull request closed without merging**